Replace comment macro with triple backticks

sifters opened the pull request:
The comment macro was replaced by Splunk with 'triple backticks' formatting in version 8.1 - which was deemed end-of-life in April 2023.
The changes included here update comment macro references in savedsearches.conf, macros.conf, and a handful of dashboards.

sifters committed:
replace comment macro woth triple backticks for single-line references

sifters committed:
replace comment macro for multi-line comments

sifters committed:
replace comment macro with triple backticks

sifters committed:
replace comment macro with triple backticks

gjanders commented:
Thanks for submitting the merge. The new splunkbase website no longer tells me which splunk versions are in use.

I suspect it's likely safe to drop support for the older versions by using the new comment style

I'll review this later this week

gjanders committed:
Update data_model_rebuild_monitor.xml

gjanders committed:
Update issues_per_sourcetype.xml

gjanders committed:
Update savedsearches.conf

Comments now work over multiple lines, removing | search when there is no longer any search
